cms/snippets/models/internal_button.py

```diff
@@ -4,59 +4,52 @@
 from wagtail.snippets.models import register_snippet
 
 
-class ButtonTypes(models.Choices):
-    """Enum for button types"""
+class InternalButtonTypes(models.Choices):
+    BULK_DOWNLOAD = "BULK_DOWNLOAD"
 
-    DOWNLOAD = "DOWNLOAD"
-    SUBMIT = "SUBMIT"
-
-
-class Methods(models.Choices):
-    """Enum for method choices"""
-
-    POST = "POST"
-    GET = "GET"
+    @staticmethod
+    def return_button_type_details() -> dict[tuple[str, str]]:
```

Comment:
This is a dict, keyed by a string with a value of tuple:
```suggestion
    def return_button_type_details() -> dict[str, tuple[str, str]]:
```
```
dict[str, tuple[str, str]]
```